src/guide/effects.mdx

```diff
@@ -0,0 +1,498 @@
+---
+title: "Graphical Effects"
+index: 7
+---
+
+@@QtQuick.Effects.MultiEffect lets you apply multiple visual effects—like blur, shadow,
+and color adjustments—in a single, easy-to-use QML component.
+
+# Qt 6 MultiEffect
+
+Now, how do I use this you ask? Well, this guide provides a few examples for common uses of
+effects such as drop shadows, colourisation, masks and blurring.
+
+First and foremost (and arguably the most commonly used), drop shadows! To create a drop shadow,
+the [QtQuick.Effects](https://doc.qt.io/qt-6/qtquick-effects-qmlmodule.html) module provides
+two components that you can use: @@QtQuick.Effects.RectangularShadow and @@QtQuick.Effects.MultiEffect.
+
+@@QtQuick.Effects.RectangularShadow is much easier to use and more efficient than
+@@QtQuick.Effects.MultiEffect, however it can only be used for rectangular shadows (as the name implies).
+On the other hand, @@QtQuick.Effects.MultiEffect can used for most common effects, including all of
+the previously mentioned effects.
+
+### Shadows
+
+The simplest way to create a rectangular drop shadow is with the @@QtQuick.Effects.RectangularShadow
+component. This component essentially creates a rectangle with blurred edges to act as a shadow. To
+use this component, you would typically create one below the source component of the shadow. To position
+a component beneath another, you can either simply create it before the other component, or use the
+@@QtQuick.Item.z property to visually position it below the other component.
+
+For example:
+
+```qml
+@@QtQuick.Rectangle {
+  id: rect
+
+  implicitWidth: 100
+  implicitHeight: 200
+  radius: 30
+}
+
+@@QtQuick.Effects.RectangularShadow {
+  anchors.fill: rect  // Fill the source rect
+  radius: rect.radius  // Have the same rounding as the source rect
+  blur: 15  // The blur/radius of the shadow
+  spread: 2  // The number of pixels from the edges of the rect the shadow extends from
+  color: "#000000"  // The colour of the shadow
+  z: -1  // Visually positions the shadow behind the source rect
+}
+```
```

Comment:
Formulate the example to have the shadow above the rectangle and don't use z index. You can still mention thats an option though.